src/app/modules/myinfo/myinfo.controller.ts

```diff
@@ -139,11 +141,17 @@ export const loginToMyInfo: ControllerHandler<
   }
   const { formId, encodedQuery } = parseStateResult.value
 
-  let redirectDestination = `/${formId}`
+  // For local dev, we need to specify the frontend app URL as this is different from the backend's app URL
+  const redirectDestinationRaw =
+    process.env.NODE_ENV === Environment.Dev
```

Comment:
iirc `Environment.Dev` refers to `staging` and friends, which might not be intended since we only want to use this local feAppUrl for local dev 🤔 

I faced this similar problem when working with `mockTwilio`. I ended up using `.env`/`.template-env` instead.

Reply:
`Environment` has 3 values - `development`, `production` and `test`. You might be referring to `isDev` (in `config.ts`) which refers to staging and friends.

I've tested this locally and on staging, on staging it redirects to the relative path :) which shows it's not covered by `Environment.Dev`

Reply:
I see noted. I've checked the workflow file in GH action for staging and it is indeed the production flavour.
```
with:
  file: ./Dockerfile.production
  ...
```